packages/flutter_tts/example/lib/main.dart

```diff
@@ -175,23 +187,27 @@ class _MyAppState extends State<MyApp> {
       future: _getLanguages(),
       builder: (BuildContext context, AsyncSnapshot<dynamic> snapshot) {
         if (snapshot.hasData) {
-          return _languageDropDownSection(snapshot.data);
+          return _languageDropDownSection(snapshot.data as List<dynamic>);
         } else if (snapshot.hasError) {
           return Text('Error loading languages...');
         } else
           return Text('Loading Languages...');
       });
 
-  Widget _inputSection() => Container(
-      alignment: Alignment.topCenter,
-      padding: EdgeInsets.only(top: 25.0, left: 25.0, right: 25.0),
-      child: TextField(
-        maxLines: 11,
-        minLines: 6,
-        onChanged: (String value) {
-          _onChange(value);
-        },
-      ));
+  Widget _inputSection() {
+    _newVoiceText = 'Hello everyone. This is a flutter tts example app.';
```

Comment:
I couldn't find this code in flutter_tts/example. Do we need it?

+) 208 line

Reply:
It was very inconvenient to have to add text every time I tested. So I added the text. 
If I have to delete the text, I will.

Reply:
We keep the frontend package's example code as much as possible for smooth version updates.
So I want to reduce unnecessary diffs as much as possible unless it is platform-specific code.
So my opinion is that it would be better to delete it.
https://pub.dev/packages/flutter_tts/example

Reply:
OK. I understood. Thank you for your review.